R/foreign.R

```diff
@@ -521,10 +521,7 @@ write_graph <- function(
 # Plain edge list format, not sorted
 ################################################################
 
-read.graph.edgelist <- function(file, n = 0, directed = TRUE, ...) {
-  if (length(list(...)) > 0) {
-    stop("Unknown arguments to read_graph (edgelist format)")
```

Comment:
the name of the function is ok. However, it might make sense to rename all those functions with underscores?